pkgs/tools/graphics/ditaa/default.nix

```
@@ -1,18 +1,18 @@
 { lib, stdenv, fetchurl, jre }:
 
-stdenv.mkDerivation rec {
-  name = "ditaa-0.11.0";
+stdenv.mkDerivation rec rec {
```

Shouldn't it be

```suggestion
stdenv.mkDerivation rec {
```

as before?